algorithms/pact/pact_ops.py

```diff
@@ -1038,6 +1046,18 @@ def forward(self, x):
         return result
 
 
+    # do not use in training!
+    def get_bias_q(self, eps_in):
+        # we assume that bias gets quantized to a really high bitwidth so don't
+        # clip it
+        with torch.no_grad():
+            b = PACTQuantize(self.bias, self.get_eps_out(eps_in).flatten(), -1000.*torch.ones_like(self.clip_lo.flatten()), 1000.*torch.ones_like(self.clip_hi.flatten()), clip_gradient=self.clip_gradient, floor=False)
+        return b
```

Comment:
-1000 and +1000 seem a bit arbitrary; maybe `2**16 * self.get_eps_out(eps_in)` or similar could also work and make this scale independent?